salesagility/SuiteCRM
# Hotfix Subpanel 'illegal string offset' error

Description

Changed variable in template to output correct value; current variable gives 'illegal string offset' error.
Added conditional wrappers where it was possible that variables were null or undefined.
Motivation and Context

Not storing or passing correct values; unnecessary error messages; potential fail in functions and results.
How To Test This

Check error logs to make sure they don't appear.
Types of changes

```
[x ] Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)
Breaking change (fix or feature that would cause existing functionality to change)
```

Final checklist

```
[x ] My code follows the code style of this project found here.
My change requires a change to the documentation.
[x ] I have read the How to Contribute guidelines.
```
